Gigs/Gigs/GigsTableViewController.swift

```diff
@@ -10,65 +10,56 @@ import UIKit
 
 class GigsTableViewController: UITableViewController {
     
-    
     let gigController = GigController()
     
-    
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        
-    }
-    
-    
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
         
         // transition to login view if conditions require
         if gigController.bearer == nil {
             performSegue(withIdentifier: "LoginViewModalSegue", sender: self)
         } else  {
-            
-        gigController.fetchAllGigs { (result) in
-            do {
-                self.gigController.gigs = try result.get()
-            } catch {
-                if let error = error as? NetworkError {
-                    switch error {
-                    case .badAuth:
-                        print("Error: Bad authorization")
-                    case .badData:
-                        print("Error: Bad data")
-                    case .decodingError:
-                        print("Error: Decoding error")
-                    case .noAuth:
-                        print("Error: No authorization")
-                    case .otherError:
-                        print("Error: Other error")
-                    case .encodingError:
-                        print("Error: Encoding error")
+            gigController.fetchAllGigs { (result) in
+                do {
+                    let success = try result.get()
+                    if success {
+                        DispatchQueue.main.async {
+                            print("Successfully fetched all gigs")
+                            self.tableView.reloadData()
```

Comment:
This makes sure that we actually reload the tableview when we get the call back from our fetch method